sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubMetricsProvider.cs

```diff
@@ -80,7 +80,7 @@ public async Task<EventHubsTriggerMetrics> GetMetricsAsync()
             }
             catch (Exception e)
             {
-                _logger.LogWarning($"Encountered an exception while checking EventHub '{_client.EventHubName}'. Error: {e.Message}");
+                _logger.LogWarning($"Encountered an exception while getting checkpoints for EventHub '{_client.EventHubName}' used for scaling. Error: {e.Message}");
```

Comment:
```suggestion
                _logger.LogWarning($"Encountered an exception while getting checkpoints for Event Hub '{_client.EventHubName}' used for scaling. Error: {e.Message}");
```

Gotta watch the branding!

Reply:
ah this was copied from a warning line above -- will change that too